src/main/java/core/basesyntax/strategy/QuantityIncrementer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.mapvalidator.KeyPresentValidator;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public interface QuantityIncrementer extends KeyPresentValidator {
```

Comment:
It is not recommended to use inheritance for interfaces unless they are semantically related. `QuantityIncrementer` should not extend `KeyPresentValidator` as their purposes are different. The `KeyPresentValidator` can be used within the method as a dependency instead of inheritance.